### PR TITLE
add configurable root password

### PR DESCRIPTION
**Pull request by bollenn**
- add configurable root password
- fix super user path

### CODE REVIEW

File: scripts/chroot.sh [hunk 1]
```diff
@@ -945,9 +945,10 @@ cat > "${DIR}/chroot_script.sh" <<-__EOF__
 				passwd -l root || true
 			else
 				passwd <<-EOF
-				root
-				root
+				${rfs_root_password}
+				${rfs_root_password}
 				EOF
+				echo "export PATH=\$PATH:/usr/local/sbin:/usr/sbin:/sbin" >> /root/.bashrc
```
Comment (bollenn) [inline scripts/chroot.sh:951]: Here am not sure this is valid for all images. At least for debian buster this helps

Comment (RobertCNelson) [reply scripts/chroot.sh:951]: last time i tried stretch, i think the path's were still messed up as root, so if it fixes buster, it should be good for stretch then...

**Comment by RobertCNelson** [comment]
Thanks all merged!

File: scripts/chroot.sh (hunk 1 above)
Comment (bollenn) [reply scripts/chroot.sh:951]: cool, thanks...

**Pull request merged**